docs/reference/service/introduction.md

```diff
@@ -24,3 +24,7 @@ likely that you'll encounter significant bugs.
 
 If you run into a bug or have a suggestion for improvement, please
 [raise an issue on GitHub](https://github.com/platformatic/platformatic/issues/new). 
+
+## Standalone usage
+
+If you're only interested in the features available in Platformatic Service, you can simply switch `platformatic` with `@platformatic/service` in the `dependencies` of your `package.json`, so that you'll only import fewer deps.
```

Comment:
```suggestion
If you're only interested in the features available in Platformatic Service, you can simply switch `platformatic` with `@platformatic/service` in the `dependencies` of your `package.json`, so that you'll only import fewer deps.

You can use the `plt-service` command, it's the equivalent of `plt service`.
```